### PR TITLE
add gdt command

### PR DESCRIPTION
**Pull request by disconnect3d**
Yet another useful command when debugging Linux kernel ;)

image

See also:
* https://wiki.osdev.org/Global_Descriptor_Table
* https://wiki.osdev.org/GDT_Tutorial

### CODE REVIEW

**Comment by patryk4815** [comment]
every new command should be added in https://github.com/pwndbg/pwndbg/tree/dev/docs/commands ?

File: pwndbg/commands/__init__.py [hunk 1]
```diff
@@ -708,6 +708,7 @@ def load_commands() -> None:
         import pwndbg.commands.dumpargs
         import pwndbg.commands.elf
         import pwndbg.commands.flags
+        import pwndbg.commands.gdt
```
Comment (mbrla0) [inline pwndbg/commands/__init__.py:711]: Just a small question. You don't seem to use `gdb` or `gdblib` at all in the command. Couldn't this import go outside the `if` for GDB-only commands?

Comment (disconnect3d) [reply pwndbg/commands/__init__.py:711]: I believe it should. I just didn't know where to put it. Please do it when u port some other commands to aglib

**Comment by disconnect3d** [comment]
> every new command should be added in https://github.com/pwndbg/pwndbg/tree/dev/docs/commands ?

Yeah we need to update docs somehow. Need to figure out how to do it again, ugh.

**Pull request merged**